client/src/main/java/io/spine/client/TargetBuilder.java

```diff
@@ -277,6 +279,22 @@ public B where(CompositeFilter... predicate) {
         return self();
     }
 
+    /**
+     * Performs the same action as {@link #where(CompositeFilter...)}, but acts as a shortcut
+     * overload for internal use.
+     *
+     * <p>If the passed set is empty, does nothing.
+     */
+    @CanIgnoreReturnValue
+    @Internal
```

Comment:
I guess we can do without the `@Internal` annotation since the method is package-private.